src/self-model.scm

```diff
@@ -308,16 +320,60 @@
 ;; Return true if low sound is heard
 (DefineLink
     (DefinedPredicate "very low sound?")
-    (NotLink (GreaterThan
+      (NotLink (GreaterThan
         (Get (State decibel-value (Variable "$y")))
         very-low-sound)))
-
+        
 ;; Return true for normal conversation
 (DefineLink
-    (DefinedPredicate  "normal conversation?")
+   (DefinedPredicate "normal conversation?")
     (NotLink (GreaterThan
         (Get (State decibel-value (Variable "$z")))
         normal-conversation)))
+        
+;; Return true if a very loud sound is heard
+(DefineLink
+	(DefinedPredicate "Heard very loud sound?")
+	(NotLink (GreaterThan
+		(Get (State decibel-value (Variable "$a")))
+		very-loud-sound)))
+;--------------------------------------------
+;;For Saliency
+(define-public salient-loc  (AnchorNode "locations"))
+(define-public initial-loc (list (NumberNode 1.0)(NumberNode 0.0)(NumberNode 0.0)))
+(State salient-loc (List initial-loc))
+
+(define salient (AnchorNode "Degree value"))
+(State salient (Number 5))
```

Comment:
ok I will. if you mean the initial degree value for the salient , it's set as 5,  (State salient (Number 5)).